### PR TITLE
LTS support, Travis and PHPunit issues revised and Github Actions

### PR DESCRIPTION
**Pull request by encreinformatique**
Following the discussion of https://github.com/FriendsOfSymfony/FOSMessageBundle/pull/352 I revised the blocking `setUp()` method. I used a new method `setUpBeforeTest()` method and commented `setUp()`.

I created the configuration for Github Actions so the tests are made from PHP 5.5 to 8.0.
The deprecation warnings make the test suite fails. So sadly, they are turned off.

I would have maintain into the `lts-support` branch you made to maintain the discussion in one place.
I hope this helps you @hex333ham 
Lastly, I would recommend ditching travis as their new plans seems against open source.

### CODE REVIEW

**Comment by hex333ham** [comment]
amazing work dude, when I have some time I'll look over this

> Lastly, I would recommend ditching travis as their new plans seems against open source.

agreed tbh, though it's more work to migrate us across to GitHub actions, even if not much more

**Comment by hex333ham** [comment]
@encreinformatique just realised I have admin access, if you're interested in being an active maintainer I can add you here, just lemme know

**Comment by GuilhemN** [comment]
That would sure be ok for me :)

Great PR indeed!

**Comment by GuilhemN** [comment]
That would sure be ok for me :)

Great PR indeed!

**Comment by encreinformatique** [comment]
Thank you @GuilhemN and @hex333ham 

To be honest, I do not use this bundle, never have. So I would not go as far as "I have more interest" than you. :)
It felt a good exercise to contribute to the bundle trying to maintain maximum compatibility with older versions.

**Comment by GuilhemN** [comment]
Ok, that's completely understandable of course and that's already very nice of you to have contributed such change :)

This can be merged for me :+1:

**Review by GuilhemN — APPROVED** [review]

**Comment by encreinformatique** [comment]
> amazing work dude, when I have some time I'll look over this but it could be a while so I'd use your fork if you can

I will not delete my fork until it is merged so do not worry about that.

> > Lastly, I would recommend ditching travis as their new plans seems against open source.
> 
> agreed tbh, though it's more work to migrate us across to GitHub actions, even if not much more

It is included in this current PR. All PHP version from `PHP 5.5` until `PHP 8.0`.
I use Github Actions more and more and Travis CI is going in a direction that I personally don't like.
The [file is there](https://github.com/encreinformatique/FOSMessageBundle/blob/lts-support/.github/workflows/ci.yml) if you need more info.

**Comment by tperunsky** [comment]
@GuilhemN 
Hi, is this PR ready to be merged, or does it require more work? I might be able to contribute to a limited extent if necessary.

**Comment by GuilhemN** [comment]
@tperunsky I can merge it, but nonetheless I would consider this bundle unmaintained and would advise to not use it if you can.

**Comment by tperunsky** [comment]
@GuilhemN  If it's no trouble then please do merge it. I understand that there is no expectation of further updates. Thank you.

**Pull request merged**
